patches/chrome-browser-notifications-notification_platform_bridge_android.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/notifications/notification_platform_bridge_android.h b/chrome/browser/notifications/notification_platform_bridge_android.h
+index 7ff354723b9d14c43b9dd7dd2679f36ab8e66f79..ffb5f27cbc4f7d5f826e4717c9a6d6914abb1949 100644
+--- a/chrome/browser/notifications/notification_platform_bridge_android.h
++++ b/chrome/browser/notifications/notification_platform_bridge_android.h
+@@ -88,6 +88,7 @@ class NotificationPlatformBridgeAndroid : public NotificationPlatformBridge {
+   static void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry);
+
+  private:
++  friend class BraveNotificationPlatformBridgeHelperAndroid;
```

Comment:
always use defines to make patches extensible

Reply:
@yachtcaptain23 ^^^

Reply:
This is only temporary until we move to custom ad notifications across all build channels. This PR is reverting previous changes so would rather not change any logic. Thanks